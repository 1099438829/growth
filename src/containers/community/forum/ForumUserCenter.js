import React, { Component } from 'react';
import { Text, View } from 'react-native';

class ForumUserCenter extends Component {
  static componentName = 'ForumUserCenter';

  render() {
    return (
      <View>
        <Text>UserCenter</Text>
      </View>
    );
  }
}

export default ForumUserCenter;
