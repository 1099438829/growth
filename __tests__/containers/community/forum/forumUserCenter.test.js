import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import ForumUserCenter from '../../../../src/containers/community/forum/ForumUserCenter';

describe('test user center', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <ForumUserCenter />,
    );

    const treeJson = tree.toJSON();
    expect(treeJson).toMatchSnapshot();
  });
});
