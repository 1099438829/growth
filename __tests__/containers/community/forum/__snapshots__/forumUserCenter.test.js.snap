// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test user center renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    UserCenter
  </Text>
</View>
`;
